packages/picongpu/picongpu.profile

```diff
@@ -7,3 +7,11 @@ spack load @PIC_SPACK_COMPILER@ >/dev/null 2>&1 || echo "using default compiler"
 
 # required: PIConGPU and its dependencies
 spack load -r @PIC_SPACK_SPEC@
+
+# activate bash completion if available
+BASH_COMP_FILE=$PICSRC/bin/picongpu-completion.bash
```

Comment:
`BASH_COMP_FILE` is quite a generic name, did we merge this to PIConGPU?
`PIC_COMP_FILE` would have been better...

Reply:
yes we merged it with this name.

Reply:
@finnolec an update of the var name (which is exposed in the user environment) would be welcome :)

Reply:
I changed it already in this PR.